addons/sys_core/initSettings.sqf

```diff
@@ -120,10 +120,10 @@
 // Reveal to AI
 [
     QGVAR(revealToAI),
-    "CHECKBOX",
+    "SLIDER",
     localize LSTRING(revealToAI_displayName),
     "ACRE2",
-    true,
+    [0, 1, 0.50, 2],
```

Comment:
```suggestion
    [0, 1, 1, 2],
```

We should have a default value of 1. This is the default value at the moment.

Reply:
Makes sense to set it realistically though, as pointed out by the issue.

Reply:
I'll go with 1. So the player's experience is the same as before